src/utils/theGraph.js

```diff
@@ -141,6 +143,21 @@ const completeQueries = {
           supportedChains[args.chainID].network,
           minionAddresses,
         );
+        const vaultData = await Promise.all(
```

Comment:
Maybe this is something we should do in the caching job eventually?

Reply:
yeah, it should be moved there